doc/classes/Projection.xml

```diff
@@ -240,6 +240,19 @@
 				Returns a [Projection] that performs the inverse of this [Projection]'s projective transformation.
 			</description>
 		</method>
+		<method name="is_equal_approx" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="to" type="Projection" />
+			<description>
+				Returns [code]true[/code] if this [Projection] and the given [Projection] are equal within a small tolerance.
```

Comment:
```suggestion
				Returns [code]true[/code] if this [Projection] and [param to] are approximately equal, by calling [method @GlobalScope.is_equal_approx] on each component.
```
To match the style in `AABB`, `Transform2D`, etc.

Reply:
It should be all right now